roles-in-tech/security.md

```diff
@@ -17,35 +18,55 @@ How the Internet Works Topic Outline
 
 ### Objectives
-**Participants will be able to:**
-- Pull a relevant JS library up to handle common scenarios
-- Validate user input
-- Authenticate users on a site
-- XSS someone else's web page
+**Participants will be able to learn:**
+- Risk Management
+- Cryptography & PKI
+- Identity and access Management
+- How to implement secure n/w architecture and system design
+- Discover security vulnerabilities across an entire network
 
 ### Specific Things To Teach
-- OWASP Secure coding practices
-	- Input validation
-	- Authentication means and pitfalls
-	- Session management
-	- Cross-site scripting (XSS)
-	- Cross-site request forgery (CSRF)
+The basic required skills as an Security Engineer are
+- Programming Skills
+- Communication Skills
+- Risk Management
+- Networking Basics
+- Situational Awareness
+- Toolkit Maintenance
 
-### Materials
+One can join course [CompTIA Security+ Certification](https://certification.comptia.org/certifications/security) or [CCNA Security](https://www.cisco.com/c/en/us/training-events/training-certifications/certifications/associate/ccna-security.html#~stickynav=1) to improve the basics of security
 
-- [Techtonica's Roles in Tech Presentation](https://drive.google.com/file/d/1oXgOX1EC4igI6ZwgJlhF6AN4dG2Yn_S4/view) (security section starts at 2:57)
-- [A quick introduction to web security [FreeCodeCamp]](https://medium.freecodecamp.org/a-quick-introduction-to-web-security-f90beaf4dd41)
-- An online certified course on [Udemy](https://www.udemy.com/certified-secure-netizen/)
--course for beginners [guru99](https://www.guru99.com/ethical-hacking-tutorials.html)
+After gaining basic skills one can go further as *Security Architect* or *Security Analyst* or *CISO* or any further specialization
 
-#### Web Security Libraries and Practices
+### Materials
 
-- [OWASP Secure Coding Practices Quick Reference Guide](https://www.owasp.org/images/0/08/OWASP_SCP_Quick_Reference_Guide_v2.pdf)
-- [Parsley, the ultimate JavaScript form validation library](http://parsleyjs.org/)
-- [Validator](https://github.com/chriso/validator.js)
-- [DOMPurify](https://github.com/cure53/DOMPurify)
-- [Passport](http://passportjs.org/)
-- [OpenID client connect](https://github.com/IdentityModel/oidc-client-js)
+- [Techtonica's Roles in Tech Presentation](https://drive.google.com/file/d/1oXgOX1EC4igI6ZwgJlhF6AN4dG2Yn_S4/view) (security section starts at 2:57)
+- online course at [CYBRAY](https://www.cybrary.it/)
+- An online certified course on [Udemy](https://www.udemy.com/courses/it-and-software/network-and-security/)
+- course for beginners [guru99](https://www.guru99.com/ethical-hacking-tutorials.html)
+- You can refer the book "[Cryptography and Network Security](https://wanguolin.github.io/assets/cryptography_and_network_security.pdf)" by William Stallings. It is a very good book for understanding the basic concepts of security
+
+### Roles and Responsibilities
+
+- developing a set of security standards and practices
```

Comment:
could you change these to "develop", "create", etc, rather than using -ing words?